internal/binary/envoy/list_test.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envoy
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/tetratelabs/getenvoy/internal/manifest"
+)
+
+func TestPrintVersions(t *testing.T) {
```

Comment:
I don't see there's a case for checking the sort is working correctly? (the order of map entries is not deterministic).

Reply:
the test has this, so it should break on non-deterministic, esp since the data is added in reverse

```go
		{
			name:     "darwin",
			goos:     "darwin",
			manifest: goodManifest,
			expected: `1.18.3
1.14.6
`,
```